### PR TITLE
Labels

### PR DESCRIPTION
**Pull request by omarkamal95**
Finished admin view of comment: 
Admin can: Remove, Add label.
#83

@KarimElSheikh Test Review.
@Veegas Code Review.
@MinaNemr Documentation Review.
@Israa-Elwan Scenario Review.
@Mernawaseif  UML Review.

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -323,26 +333,7 @@
...
-    <script src="https://static.hypothes.is/lib/annotator.min.js"></script>
-    <script src="https://static.hypothes.is/lib/annotator.document.js"></script>
-    <script src="https://static.hypothes.is/js/plugin/bridge.js"></script>
-    <script src="https://static.hypothes.is/js/plugin/heatmap.js"></script>
-    <script src="https://static.hypothes.is/js/plugin/toolbar.js"></script>
-    <script src="https://static.hypothes.is/js/inject.min.js"></script> -->
-    
-    -->
+
     <!-- build:js({.tmp,src}) scripts/scripts.js -->
```
Comment (Veegas) [inline frontend/index.html]: @omarkamal95  Sheel el 7etta dih

**Comment by Veegas** [comment]
@omarkamal95  Code review tamam bas sheel el 7eta el 3amaltelak 3aleeha comment 3ahshan hatban fel view

**Comment by Israa-Elwan** [comment]
@omarkamal95  done with reviewing the scenarios

**Comment by Mernawaseif** [comment]
@omarkamal95 done with UML review, no comments except for the uml sequence diagram, the sequence should be ordered.

**Comment by MinaNemr** [comment]
@omarkamal95 documenation reviewd
code is well documented

**Comment by KarimElSheikh** [comment]
@omarkamal95 Label tests don't pass

**Comment by omarkamal95** [comment]
@KarimElSheikh They do pass, im sure. Did you use NODE_ENV?

**Pull request merged**